examples/example_parallel_manual_spawning.py

```diff
@@ -39,6 +39,10 @@
         pass
 
 
+############################################################################
+# Define the spawner
```

Comment:
Could you please name this header 'Define utility function for multiprocessing'?